### PR TITLE
chore: cleanup migrate links and docs

### PR DESCRIPTION
**Pull request by Jolg42**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,10 +105,10 @@ The data model is a collection of [models](https://www.prisma.io/docs/reference/
...
-- Manually writing the data model and mapping it to the database with [Prisma Migrate](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-migrate)
+- Generate the data model from [introspecting](https://www.prisma.io/docs/concepts/components/introspection) a database
+- Manually writing the data model and mapping it to the database with [Prisma Migrate](https://www.prisma.io/docs/concepts/components/prisma-migrate)
 
-Once the data model is defined, you can [generate Prisma Client](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-client/generating-prisma-client) which will expose CRUD and more queries for the defined models. If you're using TypeScript, you'll get full type-safety for all queries (even when only retrieving the subsets of a model's fields).
+Once the data model is defined, you can [generate Prisma Client](h==https://www.prisma.io/docs/concepts/components/prisma-client/generating-prisma-client) which will expose CRUD and more queries for the defined models. If you're using TypeScript, you'll get full type-safety for all queries (even when only retrieving the subsets of a model's fields).
```
Comment (Jolg42) [inline README.md:111]:
```suggestion
Once the data model is defined, you can [generate Prisma Client](https://www.prisma.io/docs/concepts/components/prisma-client/generating-prisma-client) which will expose CRUD and more queries for the defined models. If you're using TypeScript, you'll get full type-safety for all queries (even when only retrieving the subsets of a model's fields).
```

File: README.md [hunk 2]
```diff
@@ -142,28 +141,28 @@ Once Prisma Client was generated, you can import in your code and send queries t
 You can import and instantiate Prisma Client as follows:
 
 ```ts
-import { PrismaClient } from '@prisma/client'
+import { PrismaClient } from "@prisma/client";
```
Comment (timsuchanek) [inline README.md:144]: Why reformat?

File: README.md [hunk 3]
```diff
@@ -182,11 +181,11 @@ const allUsers = await prisma.user.findMany({
 const filteredPosts = await prisma.post.findMany({
   where: {
     OR: [
-      { title: { contains: 'prisma' } },
-      { content: { contains: 'prisma' } },
+      { title: { contains: "prisma" } },
```
Comment (timsuchanek) [inline README.md:184]: Prettier?

File: README.md (hunk 2 above)
Comment (Jolg42) [reply README.md:144]: It just happened automatically 😅 should I revert?

File: README.md (hunk 3 above)
Comment (Jolg42) [reply README.md:184]: It just happened automatically 😅 should I revert?

File: README.md (hunk 2 above)
Comment (Jolg42) [inline README.md:144]:
```suggestion
import { PrismaClient } from "@prisma/client"
```

File: README.md [hunk 4]
```diff
@@ -142,28 +141,28 @@ Once Prisma Client was generated, you can import in your code and send queries t
 You can import and instantiate Prisma Client as follows:
 
 ```ts
-import { PrismaClient } from '@prisma/client'
+import { PrismaClient } from "@prisma/client";
 
-const prisma = new PrismaClient()
+const prisma = new PrismaClient();
```
Comment (Jolg42) [inline README.md:146]:
```suggestion
const prisma = new PrismaClient()
```

File: README.md [hunk 5]
```diff
@@ -142,28 +141,28 @@ Once Prisma Client was generated, you can import in your code and send queries t
...
 ```
 
 or
 
 ```js
-const { PrismaClient } = require('@prisma/client')
+const { PrismaClient } = require("@prisma/client");
```
Comment (Jolg42) [inline README.md:152]:
```suggestion
const { PrismaClient } = require('@prisma/client')
```

File: README.md [hunk 6]
```diff
@@ -142,28 +141,28 @@ Once Prisma Client was generated, you can import in your code and send queries t
...
 
-const prisma = new PrismaClient()
+const prisma = new PrismaClient();
```
Comment (Jolg42) [inline README.md:154]:
```suggestion
const prisma = new PrismaClient()
```

File: README.md [hunk 7]
```diff
@@ -142,7 +141,7 @@ Once Prisma Client was generated, you can import in your code and send queries t
 You can import and instantiate Prisma Client as follows:
 
 ```ts
-import { PrismaClient } from '@prisma/client'
+import { PrismaClient } from "@prisma/client"
```
Comment (Jolg42) [inline README.md:144]:
```suggestion
import { PrismaClient } from '@prisma/client'
```

File: README.md [hunk 8]
```diff
@@ -172,7 +171,7 @@ const allUsers = await prisma.user.findMany()
 // Run inside `async` function
 const allUsers = await prisma.user.findMany({
   include: { posts: true },
-})
+});
```
Comment (Jolg42) [inline README.md:174]:
```suggestion
})
```

File: README.md [hunk 9]
```diff
@@ -157,13 +156,13 @@ const prisma = new PrismaClient()
...
 
-Learn more about the available operations in the [Prisma Client API reference](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-client/api) or watch this [demo video](https://www.youtube.com/watch?v=LggrE5kJ75I&list=PLn2e1F9Rfr6k9PnR_figWOcSHgc_erDr5&index=4) (2 min).
+Learn more about the available operations in the [Prisma Client docs](https://www.prisma.io/docs/concepts/components/prisma-client) or watch this [demo video](https://www.youtube.com/watch?v=LggrE5kJ75I&list=PLn2e1F9Rfr6k9PnR_figWOcSHgc_erDr5&index=4) (2 min).
 
 ##### Retrieve all `User` records from the database
 
 ```ts
 // Run inside `async` function
-const allUsers = await prisma.user.findMany()
+const allUsers = await prisma.user.findMany();
```
Comment (Jolg42) [inline README.md:165]:
```suggestion
const allUsers = await prisma.user.findMany()
```

**Pull request merged**